docs/api/StreamStart.md

```diff
@@ -17,15 +17,40 @@ QUIC_STATUS
 
 # Parameters
 
-**TODO**
+`Stream`
+
+The valid handle to an open stream object.
+
+`Flags`
+
+The set of flags the controls the behavior of `StreamStart`:
+
+Value | Meaning
+--- | ---
+**QUIC_STREAM_START_FLAG_NONE**<br>0 | No special behavior. Executes as a blocking call, returning only after the operation has completed. The peer is not informed of the stream starting until the app sends data.
+**QUIC_STREAM_START_FLAG_FAIL_BLOCKED**<br>1 | Only opens the stream if flow control (from the peer) allows. If the stream count limit has currently been reached the start will fail with `QUIC_STATUS_STREAM_LIMIT_REACHED`.
+**QUIC_STREAM_START_FLAG_IMMEDIATE**<br>2 | Indicates that the peer should be immediately informed of the stream opening (or at least as soon as flow control allows) and not wait for any data to be sent first.
+**QUIC_STREAM_START_FLAG_ASYNC**<br>4 | Causes the API to be non-blocking. The API call itself will just queue the operation and immediate return. The final result of the start operation will be delivered via the `QUIC_STREAM_EVENT_START_COMPLETE` event.
+**QUIC_STREAM_START_FLAG_SHUTDOWN_ON_FAIL**<br>8 | Causes the stream to immediate shutdown (abortive) if the start operation fails.
```

Comment:
Why would I want to use this?

Reply:
I can simply the failure scenarios so the app doesn't have to do anything special.

Reply:
So then, why would I not want to use this?

Reply:
You might not want to shutdown the stream on failure because you may want to reuse it and try again.

Reply:
Is that semantically any different than shutting down the stream and opening a new stream?

Reply:
No. But it's more efficient.